.github/workflows/codecov.yml

```diff
@@ -0,0 +1,114 @@
+name: Codecov
+on:
+  push:
+    branches: [ master, ci_github ]
+
+jobs:
+  run:
+    name: test ${{ matrix.os }} / ${{ matrix.python }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        python: ["3.8","3.9","3.10","3.11"]
+    env:
+      OS: ${{ matrix.os }}
+      PYTHON: ${{ matrix.python }}
+    steps:
+    - uses: actions/checkout@master
+
+    - name: create .env file
+      shell: bash
+      run: |
+        cat <<EOF>.env
+        ACME_TESTS_PG_PW=acme-broker-debug-pw
+        ACME_SUPERUSER_PW=YOUR_SUPERUSER_PW
+        ACME_ADMIN_PW=YOUR_ADMIN_PW
+        ACME_RW_PW=YOUR_READ_WRITE_PW
+        ACME_RO_PW=YOUR_READ_ONLY_PW
+        ACME_PREFIX=/FIXME
```

Comment:
/home/acme ?

Reply:
currently ACME_PREFIX is not used in the CI - it relates to the docker-compose.yml, will remove.